website/docs/r/spring_cloud_java_deployment.html.markdown

```diff
@@ -66,6 +66,8 @@ The following arguments are supported:
 
 * `spring_cloud_app_id` - (Required) Specifies the id of the Spring Cloud Application in which to create the Deployment. Changing this forces a new resource to be created.
 
+* `active` - (Optional) should the Deployment be active?
```

Comment:
it doesn't look like this can be set?

```
------- Stdout: -------
=== RUN   TestAccSpringCloudJavaDeployment_update
=== PAUSE TestAccSpringCloudJavaDeployment_update
=== CONT  TestAccSpringCloudJavaDeployment_update
    testcase.go:110: Step 3/6 error: Error running apply: exit status 1
        
        Error: updating Spring Cloud Deployment: (Deployment Name "acctest-scjdseumb" / App Name "acctest-sca-220627191003736675" / Spring Name "acctest-sc-220627191003736675" / Resource Group "acctestRG-spring-220627191003736675"): appplatform.DeploymentsClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="BadArgument" Message="Invalid arguments: properties.active is not supported to update." Details=[{"code":"BadArgument","details":null,"message":"properties.active is not supported to update.","target":"properties.active"}]
        
          with azurerm_spring_cloud_java_deployment.test,
          on terraform_plugin_test.tf line 25, in resource "azurerm_spring_cloud_java_deployment" "test":
          25: resource "azurerm_spring_cloud_java_deployment" "test" {
        
--- FAIL: TestAccSpringCloudJavaDeployment_update (571.49s)
FAIL
```